File: Queue/queueTest.cpp
#include "Queue.h"

int main(int argc, char* argv[])
{
    Queue<int> queue;
    for (int i = 0; i < 20; i++)
        queue.enQueue(random() % 100);
    int value = queue.deQueue();
    cout << value << endl; 
    
    return 0; 
}

File: java/datastructure/datastructure/testing/QueueTest.java
package datastructure.testing;

import datastructure.Queue.Queue; 


public class QueueTest<T> {
    public static void main(String[] args) {
        Queue<Integer> queue = new Queue<Integer>();
        // queue testing. 
        for(int i = 0; i < 20; i++) {
            queue.enQueue((int)(Math.random() * 200)); 
        }
        int data = -1; 
        queue.printQueue(); 
        while(!queue.isEmpty()) {
            data = queue.deQueue(); 
            System.out.println(data); 
        }
        

    }
}
